hack/release.sh

```diff
@@ -2,21 +2,26 @@
 
 set -e
 
+RELEASE_TOOL_IMAGE="ghcr.io/talos-systems/release-tool:latest"
+
+function release-tool {
+  docker pull "${RELEASE_TOOL_IMAGE}" >/dev/null
```

Comment:
Why not `-q` / `--quiet`?

Reply:
:+1: I'll get it updated

Reply:
nope, it doesn't work, still prints the image name, so I'll keep it this way. It still prints the errors though, so it's harmless